lib/manager/poetry/artifacts.spec.ts

```diff
@@ -3,12 +3,14 @@ import _fs from 'fs-extra';
 import { join } from 'upath';
 import { envMock, mockExecAll } from '../../../test/exec-util';
 import { loadFixture, mocked } from '../../../test/util';
+import { setAdminConfig } from '../../config/admin';
 import * as _datasource from '../../datasource';
 import { setExecConfig } from '../../util/exec';
 import { BinarySource } from '../../util/exec/common';
 import * as docker from '../../util/exec/docker';
 import * as _env from '../../util/exec/env';
 import * as _hostRules from '../../util/host-rules';
+import { UpdateArtifactsConfig } from '../types';
```

Comment:
```suggestion
import type { UpdateArtifactsConfig } from '../types';
```

please also check the other imports